pkg/syncer/metadatasyncer.go

```diff
@@ -569,31 +569,30 @@ func pvUpdated(oldObj, newObj interface{}, metadataSyncer *metadataSyncInformer)
 		log.Debugf("PVUpdated: PV %s metadata is not updated since updated PV is in phase %s", newPv.Name, newPv.Status.Phase)
 		return
 	}
-	// Verify if csi migration is ON and check if there is any label update or migrated-to annotation was received for the PV
-	if newPv.Spec.VsphereVolume != nil {
-		if metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.CSIMigration) {
-			if (oldPv.Status.Phase == v1.VolumeAvailable || oldPv.Status.Phase == v1.VolumeBound) &&
-				reflect.DeepEqual(newPv.GetAnnotations(), oldPv.GetAnnotations()) &&
-				reflect.DeepEqual(newPv.Labels, oldPv.Labels) {
-				log.Debug("PVUpdated: PV labels and annotations have not changed")
+	migrationEnabled := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.CSIMigration)
+	if migrationEnabled && newPv.Spec.VsphereVolume != nil {
+		if !migratedToAnnotationExists(ctx, newPv.Annotations) {
+			log.Info("migration annotation is not present on the PV. Skipping update")
```

Comment:
add "PVUpdated" in the beginning of log message

Reply:
done.